pkg/cluster/api/pdapi.go

```diff
@@ -345,13 +346,23 @@ func (pc *PDClient) EvictStoreLeader(host string, retryOpt *utils.RetryOption) e
 		break
 	}
 
-	if latestStore == nil || latestStore.Status.LeaderCount == 0 {
+	if latestStore == nil {
+		return nil
+	}
+
+	// XXX: the status address in store will be something like 0.0.0.0:20180
+	statusAddress := strings.Split(latestStore.Store.Address, ":")
+	statusAddress[1] = strings.Split(latestStore.Store.StatusAddress, ":")[1]
+	var leaderCount int
+	if leaderCount, err = countLeader(strings.Join(statusAddress, ":")); err != nil {
```

Comment:
If there's no default implementation of `countLeader()`, I think we don't have to pass it from argument.

Reply:
i pass it from argument is because that it's the tikv's logic and it's not good to access tikv in pdapi.go